### DISCUSSION TITLE
Временная таблица с автозаполнением и деревом

### DISCUSSION THREAD

**Discussion by AlexDeadly** [totumonline/totum-ru-issues-and-discussions#452] [Как это сделать?] [answered]
Добрый день.
Пытаюсь реализовать добавление товаров в заказ списком как в примере, только справочник товаров у меня в виде дерева. То есть есть отдельная простая таблица с категориями (папками) и есть другая таблица с товарами, в котором с помощью поля "tree" реализовано дерево как было показано в уроке.

Можно ли во временной таблице одновременно использовать поля "insert" (для автозаполнения) и "tree" (для формирования дерева)? Пока не удалось заставить работать оба поля одновременно. Или такое нужно реализовывать каким-то другим способом?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
<img width="592" alt="Снимок экрана 2023-10-13 в 13 48 07" src="https://github.com/totumonline/totum-ru-issues-and-discussions/assets/55755565/7ebbf766-3e8b-4508-9868-abb64066661a">

<img width="594" alt="Снимок экрана 2023-10-13 в 13 48 21" src="https://github.com/totumonline/totum-ru-issues-and-discussions/assets/55755565/1764f14e-b893-4687-a3d6-e2af84cc6649">

1. Дерево с типом По другой таблице тоже возможно. 
2. insert должен быть первым полем в табл по sort, а для tree это не обязательно

**Reply by ttm-alexeypolunin** [reply]
Если необходимо показать на вашем проекте — приходите на платную консультацию, сделаем.

**Reply by AlexDeadly** [reply]
Понял. Спасибо, видимо что-то недонастроил. Делал вариант когда insert - первое поле, а tree - второе, но дерево почему-то не отображалось. Разберусь в чем дело и напишу сюда дополнительную информацию.
